commit bad37bf86798d30292769581aca7e71471c34485
Author: Matthew Toseland <toad@amphibian.dyndns.org>
Date:   Fri Nov 5 23:30:43 2010 +0000

    Use the specific, absolute path, rather than loading from path, for NativeBigInteger library.

diff --git a/src/freenet/support/LibraryLoader.java b/src/freenet/support/LibraryLoader.java
index d4a22a0..2470d40 100644
--- a/src/freenet/support/LibraryLoader.java
+++ b/src/freenet/support/LibraryLoader.java
@@ -45,7 +45,7 @@ public class LibraryLoader {
 		File nativeLib = new File((System.getProperty("java.library.path")) + "/lib" + libraryName + prefix);
 		if (nativeLib.exists()) {
 			System.out.println("Attempting to load the NativeThread library ["+libraryName+']');
-			System.loadLibrary(libraryName);
+			System.loadLibrary(nativeLib.getAbsolutePath());
 			success = true;
 		} else {
 			try {
